index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <header class="images">
+    <a href="#"><img src="images/MyLogo.png" alt="my logo"></a>
+    <nav>
+      <img src="images/Union.png" alt="union">
+    </nav>
+    </header>
+
+ <section id="headline">
+    <div id="headline-content">
+      <h1 class="header">I'm Khadar, <br> Glad to see you!</h1>
+      <p class="portfolio-description">I am a software developer! I can help you build a product , feature or website
+        Look through some of my work and experience. If you like what you see and have a project you need coded, don't hestiate to contact me!
+      </p>
+      <h2 class="lets-connect">LET'S CONNECT</h2>
+
+      <ul class="social-media-links">
```

Comment:
Great job @Khadaro 🤝

- [ ] Just like the previous reviewer stated. Kindly remove both the horizontal and vertical scrolls from your design.
![scrolls](https://user-images.githubusercontent.com/63917725/185676146-4d295c64-a5e4-4677-a7ac-c68bb2981878.PNG)